app/ante/fee_checker.go

```diff
@@ -0,0 +1,68 @@
+package ante
+
+import (
+	errors "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerror "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+const (
+	// priorityScalingFactor is a scaling factor to convert the gas price to a priority.
+	priorityScalingFactor = 1_000_000
+)
+
+// checkTxFeeWithValidatorMinGasPrices implements the default fee logic, where the minimum price per
+// unit of gas is fixed and set by each validator, can the tx priority is computed from the gas price.
```

Comment:
```suggestion
// unit of gas is fixed and set by each validator, where the tx priority is computed from the gas price.
```
maybe?

Reply:
can I fix this is a follow @SweeXordious so we can merge this today and begin backporting where needed?